core-services/pdc/src/main/java/org/silverpeas/core/pdc/pdc/service/DefaultPdcClassifyManager.java

```diff
@@ -141,26 +132,6 @@ public void deletePosition(int nPositionId, String sComponentId) throws PdcExcep
       // Get all the positions for the given silverObjectId
       List<Position> positions = classifyEngine.findPositionsBySilverOjectId(silverObjectId);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "positions". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1488)